package.json

```diff
@@ -5,16 +5,17 @@
   "main": "index.js",
   "private": true,
   "scripts": {
-    "build": "./scripts/build.sh",
+    "build": "ng-packagr -p packages/nx/ng-package.json && ./scripts/build.sh",
```

Comment:
previously we had the build and package commands. The first one would compile things and copy statics, the second one would make them ready to package.

The build command has changed to do the latter. Having the two commands in the way they are written now doesn't feel right. We either need to move the packaging functionality from build to package, or merge the two commands.